test/interpreter_functional/test_suites/run_pipeline/basic.js

```diff
@@ -97,16 +95,17 @@ export default function ({ getService, updateBaselines }) {
         // we reuse that response to render 3 different charts and compare screenshots with baselines
         const tagCloudExpr =
           `tagcloud metric={visdimension 1 format="number"} bucket={visdimension 0}`;
-        await expectExpression('partial_test_1', tagCloudExpr, context).toMatchScreenshot();
+        await (await expectExpression('partial_test_1', tagCloudExpr, context).toMatchSnapshot()).toMatchScreenshot();
 
         const metricExpr =
-          `metricVis metric={visdimension 1 format="number"} bucket={visdimension 0}`;
-        await expectExpression('partial_test_2', metricExpr, context).toMatchScreenshot();
+          `metricVis metric={visdimension 1 format="number"} bucket={visdimension 0} colorRange={range from=0 to=10000}`;
```

Comment:
I had to specifically pass colorRange, otherwise metricVis renderer failed in runtime